aviatrix/resource_aviatrix_vpn_cert_download.go

```diff
@@ -0,0 +1,100 @@
+package aviatrix
+
+import (
+	"fmt"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/terraform-providers/terraform-provider-aviatrix/goaviatrix"
+)
+
+func resourceAviatrixVPNCertDownload() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceAviatrixVPNCertDownloadCreate,
+		Read:   resourceAviatrixVPNCertDownloadRead,
+		Update: resourceAviatrixVPNCertDownloadCreate,
+		Delete: resourceAviatrixVPNCertDownloadDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"download_enabled": {
+				Type:        schema.TypeBool,
+				Default:     false,
+				Optional:    true,
+				Description: "Whether the VPN Certificate download is enabled `gw_name`. Supported Values: \"true\", \"false\"",
+			},
+			"saml_endpoints": {
+				Type:     schema.TypeList,
```

Comment:
Does the order of these endpoints matter? If not, this should be a TypeSet

Reply:
The order doesn't matter. Changed it to a set